homeassistant/config_entries.py

```diff
@@ -362,9 +382,18 @@ def async_entries(self, domain=None):
             await async_setup_component(
                 self.hass, entry.domain, self._hass_config)
 
+        # remove notification if 0 flows with discovery source in progress
```

Comment:
This comment sounds weird looking at the code below it. I don't get it.

Reply:
I like the idea behind this code and make it faster but the comment was wrong

Reply:
When we finish the last config flow that has source=discovery, we should remove the notification.

Reply:
Yes, but that's not what the code is doing. This comment was a misplaced duplicate from the one below.